README.md

```diff
@@ -171,11 +171,7 @@ Loops are a complicated relic of archaic programming languages. In WhenWillProgr
 
 ## Installation
 
-To install WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite to your command line, first install the WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite installer.<br>
-To install the WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite installer, install the WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite installer installer.
-
-**New for 2022!**<br>
-Due to the complicated installation process, you can now install the 'Create WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite App' app that installs everything for you!
+To listen to this part [download the .mp3 file](files/installation.mp3)
```

Comment:
```suggestion
To listen to this part ![download the .mp3 file](files/installation.mp3)
```

Reply:
Ack seems like that still didn't do it

Reply:
Ah
<img width="626" alt="image" src="https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/15892272/9cf717a6-d967-4acb-a2d2-40e9503a9266">

Maybe it's worth converting it into a video file. then people can just click play

Reply:
![installation.webm](https://github.com/TodePond/WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite/assets/57299889/02d3961e-0fc3-4f77-8493-53e32310677d)